images/linux/ubuntu2204.pkr.hcl

```diff
@@ -379,6 +379,11 @@ build {
     scripts           = ["${path.root}/scripts/base/reboot.sh"]
   }
 
+  provisioner "shell" {
+    execute_command = "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+    inline          = ["cat /proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal", "sysctl -a | grep nf_conntrack_tcp_be_liberal"]
+  }
+
```

Comment:
Only temporary to verify if the `net.netfilter.nf_conntrack_tcp_be_liberal` change sticks after reboot.

Reply:
The change didn't stick 😞 https://github.com/actions/runner-images/actions/runs/5556097720/jobs/10148164916?pr=7860#step:7:17962
```
==> azure-arm.build_vhd: Provisioning with shell script: /home/vsts/Agents/image-generation/_work/runner-images/runner-images/images/linux/scripts/base/reboot.sh
    azure-arm.build_vhd: Reboot VM
==> azure-arm.build_vhd: Provisioning with shell script: /tmp/packer-shell1639431296
    azure-arm.build_vhd: 0
    azure-arm.build_vhd: net.netfilter.nf_conntrack_tcp_be_liberal = 0
```